site/frontend/src/perfetto.ts

```diff
@@ -39,3 +39,9 @@ function openTrace(arrayBuffer: ArrayBuffer, title: string) {
 
   window.addEventListener("message", onMessageHandler);
 }
+
+export const checkIsEmbeddedPerfettoEnabled = async (): Promise<boolean> => {
```

Comment:
This is a bit head-scratching, could you just make an async function please?
```typescript
export async function checkIsEmbeddedPerfettoEnabled(): Promise<boolean> {
   const status = await fetch(`/perfetto/index.html`);
   return status === 200;
} 
```

Reply:
I could do that, but in modern web frontend development, I think const arrow functions are preferred for several reasons. Which do you prefer?

Reply:
I mean, I'm not primarily a web developer, so I don't follow all the trends :laughing: Arrow functions are surely useful for inline callbacks and they can be useful for object methods because they have a more reasonable behavior w.r.t. `this`. But this seems like a perfectly normal function, so I don't see any reason why it should be an arrow function.

Most importantly, everywhere else in the code we use normal functions for such helper code, so it would be inconsistent :laughing:

Reply:
Sorry for the late. I was bit swamped last week. Alright, I'll rewrite it.